package.json

```diff
@@ -41,8 +41,10 @@
     "bootstrap-vue": "^2.21.2",
     "bs58": "^4.0.1",
     "chart.js": "^4.3.0",
+    "clipboard": "^2.0.11",
```

Comment:
We already use v-clipboard, can you re-use that instead of a new/this dependency?